ProjectLighthouse/Migrations/20220205132152_CommentRefactor.cs

```diff
@@ -0,0 +1,114 @@
+﻿using LBPUnion.ProjectLighthouse;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace ProjectLighthouse.Migrations
+{
+    [DbContext(typeof(Database))]
+    [Migration("20220205132152_CommentRefactor")]
+    public partial class CommentRefactor : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Comments_Users_TargetUserId",
+                table: "Comments");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Comments_TargetUserId",
+                table: "Comments");
+
+            migrationBuilder.RenameColumn(
+                name: "TargetUserId",
+                table: "Comments",
+                newName: "Type");
```

Comment:
Does this migrate existing comments properly? This doesn't seem right...

Reply:
No it doesn't, good catch.